includes/html/graphs/device/storage.inc.php

```diff
@@ -6,7 +6,7 @@
 require 'includes/html/graphs/common.inc.php';
 
 $iter         = '1';
-$rrd_options .= " COMMENT:'                    Size      Used    %age\\l'";
+$rrd_options .= " COMMENT:'                        Size      Used    Ratio\\l'";
```

Comment:
Shouldn't it be "% Used", like the other graphs?

Reply:
@Jellyfrog i changed it to "% Used"